Wendy.jiang/update event management doc (#22900)
DataDog/documentation: content/en/service_management/events/correlation/triage_and_notify.md

From the [Correlation][1] page, find the pattern you want to analyze and click **Triage Cases** at the end of the same row. You can also click **View All Cases** at the top of the page to view all cases with correlated events in [Case Management][2]. Datadog pulls in related metrics and logs so you can troubleshoot issues with all the related data in one place.

## Event Management Case

| Feature | Description | 
| ------  | ----------- | 
| Priority | highest priority of correlated alerts |
| Attribute | tags from correlated events. user updates won't get overriden by the engine |

> | Status | automatically managed by system, user updates will get overriden by system. Case will auto resolve when all of the underline alerts recover and automatically reopen when any alert re-trigger during the maximum alive timewindow |

Reviewer: What is the system? 

```suggestion
| Status | automatically managed by system, user updates will get overriden by system. Cases will auto resolve when all of the alerts recover and automatically reopen when any alert is triggered during the maximum alive time window |
```

> | Deletion | select the checkbox on the alert to delete any irrelavant alerts, deleted alerts won't get correlated again |

Reviewer: ```suggestion
| Deletion | select the checkbox on the alert to delete any irrelevant alerts, deleted alerts won't get correlated again |
```

> | Enriched Alerts | some cases will get automatically enriched with intelligent alerts that Datadog thinks are related based on your infrastructure. enriched alert doesn't impact case attribute, priority and status |

Reviewer: ```suggestion
| Enriched Alerts | some cases will get automatically enriched with intelligent alerts that Datadog thinks are related based on your infrastructure. Enriched alerts do not impact case attribute, priority, and status |
```

> **Investigation**

Reviewer: ```suggestion
### Investigation
```

1. From the case Overview, click **Investigation**
1. Under the *Correlations* section, you can see a list of alerts and events
1. Click into any of the alerts or events to view all related metrics and logs in context of the alert
1. (Optional) Select any alerts or events you want to remove that are not related to the case
1. Under the *Related Metrics* section, compare all related metrics or group by tags

> Check [Case Management documentation][5] for more general case management operations.

Reviewer: ```suggestion
For more information on Case Management operations, see the [Case Management documentation][5].
```
